### PR TITLE
Improvise dhclient command and add HW addr in management interface

### PR DESCRIPTION
**Pull request by ypise**
Signed-off-by: Yashodhan Pise yashodhan.pise@seagate.com

### CODE REVIEW

File: srv/components/system/network/prepare.sls [hunk 1]
```diff
@@ -43,18 +43,20 @@ Stop and disable NetworkManager service:
 # If not killed explicitly, it causes network restart to fail: COSTOR-439
 Kill dhclient:
   cmd.run:
-    - name: pkill -SIGTERM dhclient
+    # - name: pkill -SIGTERM dhclient
+    - name: dhclient -x
```
Comment (andkononykhin2) [inline srv/components/system/network/prepare.sls:47]: @ypise  as long as we will start dhclient with particular iface don't you think we need to specify iface here as well (not sure how self kill relates with that actually) ?

Comment (ypise) [reply srv/components/system/network/prepare.sls:47]: We probably don't need an interface here, but I'm wondering what happens if there are one than one processes running.

Comment (ypise) [reply srv/components/system/network/prepare.sls:47]: Agreed, we need to only change the value for one interface, but it doesn't work.
So we need to go back to pkill.
I'll prefer to disable the kill and restart logic as it doesn't make sense anymore.

File: srv/components/system/network/prepare.sls [hunk 2]
```diff
@@ -41,28 +41,30 @@ Stop and disable NetworkManager service:
...
+# network_manager_is_not_running:
+#   test.nop: []
 
-{% endif %}
+{#% endif %#}
 
-# Dummy placeholder for network.prepare:
-#   test.show_notification:
-#     - text: "A yaml file with comments results in minion non-zero exit"
+Dummy placeholder for network.prepare:
```
Comment (83bhp) [inline srv/components/system/network/prepare.sls:68]: Instead we can skip running prepare state from init.

**Review by 83bhp — APPROVED** [review]
approving with minor comment, rest looks ok.

File: srv/components/system/network/prepare.sls (hunk 2 above)
Comment (ypise) [reply srv/components/system/network/prepare.sls:68]: This is necessary as there are multiple calls internally and there's possibility of direct call somewhere.
But, I've commented it in init.sls as well

**Pull request merged**